Sources/Interaction/Manipulators/MouseCameraTrackballFirstPersonManipulator/index.js

```diff
@@ -130,6 +88,9 @@ function vtkMouseCameraTrackballFirstPersonManipulator(publicAPI, model) {
     interactor.requestAnimation(animationRequester);
     animationSub = interactor.onAnimation(() => performStep());
   };
+
+  //----------------------------------------------------------------------------
+  publicAPI.endInteraction = () => !internal.interactor.isPointerLocked();
```

Comment:
Please add a comment to explain what's going on "In pointer lock mode, manipulator should stay active even after a mouse button release".